src/pages/Product/ProductDetail/Detail.js

```diff
@@ -0,0 +1,126 @@
+import React, { useState } from 'react';
+import { Link } from 'react-router-dom';
+import './Detail.scss';
+import { FaRegHeart, FaRegStar } from 'react-icons/fa';
+import { FiPlus } from 'react-icons/fi';
+
+const Detail = () => {
+  return (
+    <div className="ProductDetail">
```

Comment:
컴포넌트 명 = 최상위 태그 명 으로 맞춰주세요. 이름은 동일하게 맞춰주지만, 팀에서 사용하신 네이밍 컨벤션이 카멜케이스면 최상위 태그 명도 동일하게 맞춰주셔야 합니다. 
```suggestion
    <div className="detail">
```